schema-derivatives-perpfutures.graphql

```diff
@@ -63,14 +63,14 @@ enum RewardTokenType {
 
   " For reward tokens awarded to borrowers "
   BORROW
-  
+
   " For reward tokens awarded to stakers "
   STAKE
 }
 
-type RewardToken @entity (immutable: true) {
+type RewardToken @entity(immutable: true) {
   " { Reward token type }-{ Smart contract address of the reward token } "
-  id: Bytes!
+  id: String!
```

Comment:
Take a look at this one: https://github.com/messari/subgraphs/blob/master/schema-bridge.graphql#L172

We can keep it as bytes, and consistent with bridges

Reply:
fixed at 3864986b411c56a979d1bde345c6a0abe72c0402 Thanks!